packages/@lwc/engine-dom/src/apis/lightning-element.ts

```diff
@@ -33,12 +33,11 @@ function getCustomElementConstructor(Ctor: ComponentConstructor): HTMLElementCon
 
 /**
  * This static getter builds a Web Component class from a LWC constructor so it can be registered
- * as a new element via customElements.define() at any given time. E.g.:
- *
- *      import Foo from 'ns/foo';
- *      customElements.define('x-foo', Foo.CustomElementConstructor);
- *      const elm = document.createElement('x-foo');
- *
+ * as a new element via customElements.define() at any given time.
+ * @example
+ * import Foo from 'ns/foo';
+ * customElements.define('x-foo', Foo.CustomElementConstructor);
+ * const elm = document.createElement('x-foo');
```

Comment:
Can we put this in a markdown code block so IDEs show it nicely formatted?

Reply:
VS Code shows it nicely formatted without code blocks, are there other IDEs that don't?
![image](https://github.com/salesforce/lwc/assets/62956339/bac30f87-5c99-4a38-b5c5-a8bb8053a9c5)

Reply:
Shows what I know! WebStorm shows it nicely formatted too:

![Screenshot 2024-02-22 at 1 19 06 PM](https://github.com/salesforce/lwc/assets/283842/1d268bba-c3c0-44ce-a0d8-99216b37a583)

Reply:
Neat!